runtime/pyproject.toml

```
@@ -13,7 +13,8 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.6.2"
-numpy = "^1.19.0, != 1.21.5"
+# use numpy that has whl for 32bit arm.
+numpy = "^1.19.0, < 1.21.5"
```

```suggestion
numpy = [
    {version = "^1.19.0, < 1.21.5", python = "< 3.8"},
    {version = "^1.19.0", python = ">= 3.8}
]
```

See: 

- https://www.piwheels.org/project/numpy/
- https://python-poetry.org/docs/dependency-specification/#multiple-constraints-dependencies